app/src/main/java/com/android/vending/billing/utils/Security.java

```diff
@@ -47,7 +47,7 @@
     /**
```

Comment:
This is a third party library, prefer not to make changes here, as it makes it difficult to merge in future changes to this library.